core/src/main/java/com/sequenceiq/cloudbreak/service/stack/CloudResourceAdvisor.java

```diff
@@ -166,6 +177,56 @@ private boolean fallbackGatewayFilter(String hostGroupName) {
                 || lowerName.contains("services");
     }
 
+    private ResizeRecommendation recommendResize(BlueprintTextProcessor blueprintTextProcessor) {
+        ResizeRecommendation resizeRecommendation = blueprintTextProcessor.recommendResize();
+
+        if (resizeRecommendation.getScaleUpHostGroups().isEmpty()) {
+            Set<String> scaleUpHostGroups = filterHostGroupByPredicate(blueprintTextProcessor, this::fallbackScaleUpFilter);
+            if (!scaleUpHostGroups.isEmpty()) {
+                resizeRecommendation.setScaleUpHostGroups(scaleUpHostGroups);
+            }
+        }
+        if (resizeRecommendation.getScaleDownHostGroups().isEmpty()) {
+            Set<String> scaleDownHostGroups = filterHostGroupByPredicate(blueprintTextProcessor, this::fallbackScaleDownFilter);
+            if (!scaleDownHostGroups.isEmpty()) {
+                resizeRecommendation.setScaleDownHostGroups(scaleDownHostGroups);
+            }
+        }
+
+        return resizeRecommendation;
+    }
+
+    private boolean fallbackScaleUpFilter(String hostGroupName) {
+        String lowerCaseName = hostGroupName.toLowerCase();
+        return !lowerCaseName.contains("master")
+                && !lowerCaseName.contains("manager");
+    }
+
+    private boolean fallbackScaleDownFilter(String hostGroupName) {
+        String lowerCaseName = hostGroupName.toLowerCase();
+        return !lowerCaseName.contains("master")
+                && !lowerCaseName.contains("manager");
+    }
+
+    private AutoscaleRecommendation recommendAutoscale(BlueprintTextProcessor blueprintTextProcessor) {
+        AutoscaleRecommendation autoscaleRecommendation = blueprintTextProcessor.recommendAutoscale();
+
+        if (autoscaleRecommendation.getHostGroups().isEmpty()) {
+            Set<String> autoscaleGroups = filterHostGroupByPredicate(blueprintTextProcessor, this::fallbackAutoscaleFilter);
+            if (!autoscaleGroups.isEmpty()) {
+                autoscaleRecommendation = new AutoscaleRecommendation(autoscaleGroups);
+            }
+        }
+
+        return autoscaleRecommendation;
+    }
+
+    private boolean fallbackAutoscaleFilter(String hostGroupName) {
+        String lowerCaseName = hostGroupName.toLowerCase();
+        return lowerCaseName.contains("compute")
+                || lowerCaseName.contains("gateway");
+    }
```

Comment:
please include only compute.

Reply:
Split it into time and load. Included only compute for load.

Reply:
👍